templates/webapi/main/index.html.ep

```diff
@@ -80,6 +80,25 @@
                 <%= help_popover('Help for <i>Full screen</i>' => '<p>Show builds in full screen mode</p>')
                 %>
             </div>
+
+            <div class="form-group" id="filter-interval-content">
+                <strong>
+                    Auto refresh
+                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically</p>') %>
+                </strong>
+
+                <input value="0" name="interval" type="radio" id="filter-interval_0">
```

Comment:
Can we avoid mixing hyphens and underscores? I don't care that much but having both is confusing.